### DISCUSSION TITLE
1.12.2

### DISCUSSION THREAD

**Discussion by jimmyG1** [maruohon/litematica#768] [Q&A] [answered]
is there a 1.12.2 version of this mod as it is a good mod and i would love to use it in 1.12.2

**Comment by maruohon** [comment, marked as answer]
Yes, all the main development still happens in the 1.12.2 LiteLoader branch of the mod. The 1.12.2 versions are available from here for now: https://masa.dy.fi/mcmods/client_mods/?mcver=1.12.2

The LiteLoader version is the most recent... but I'm guessing you are probably using Forge? Then you would be better off using the Forge version, although it's a based on older code currently.

Depending on what other mods you have, if you get a crash that mentions a SecurityException, then you might need to use a separate version of malilib without the Mixin library embedded. That's currently here: https://masa.dy.fi/tmp/minecraft/mods/misc_test_versions/malilib-forge-1.12.2-0.10.0-dev.23.nomixin.jar

**Reply by jimmyG1** [reply]
will the forge versions be getting some love evishly

**Reply by maruohon** [reply]
Yes, I'll update them again from the LiteLoader branch once I hit a "good spot" in my code rewrites and clean-up.
